apps/console/src/public/resources/applications/assets/images/illustrations/mcp-template-illustration.svg

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Let's update the year

Reply:
Will address this with a follow up PR, due to urgency.